.grabl/automation.yml

```diff
@@ -122,6 +122,9 @@ build:
         bazel test //test/assembly:docker --test_output=streamed
     deploy-artifact-snapshot:
       image: vaticle-ubuntu-21.04
+      filter:
+        owner: vaticle
+        branch: [master, development]
```

Comment:
Our Nexus repo became rapidly cluttered with `typedb` artifact snapshots in the last 2 months, and ran out of disk space.

To fix the issue, we're adding this filter to prevent the job from running on people's forks of the repo, and on other branches.

According to the Grabl source code, `filter.branch` can be either a `string` or `list`. We conclude that this syntax means the job will run if the branch is either named "master" or "development".